openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -177,6 +179,19 @@ def add_credit_course(self, course_key=None, enabled=True):
 
         return credit_course
 
+    def create_and_enroll_user(self, username, password, course_id=None, mode=CourseMode.CREDIT_MODE):
+        """ Create and enroll the user in the given course's and given mode."""
+        if course_id is None:
+            course_id = self.course_key
+
+        user = UserFactory.create(username=username, password=password)
+        self.enroll(user, course_id, mode)
+        return user
+
+    def enroll(self, user, course_id, mode):
```

Comment:
@adampalay Cant find such method in [ModuleStoreTestCase](https://github.com/edx/edx-platform/blob/8b1978ba6c8408fa6bb3fce59d363ccf5bd91c99/common/lib/xmodule/xmodule/modulestore/tests/django_utils.py#L344-344).

Am I missing something?